strings/aho_corasick.py

```diff
@@ -38,7 +47,7 @@ def add_keyword(self, keyword: str) -> None:
         self.adlist[current_state]["output"].append(keyword)
 
     def set_fail_transitions(self) -> None:
-        q = deque()
+        q: Deque = deque()
```

Comment:
Deque[dict]

Reply:
Actually the deque is storing [int], not [dict], so that's why it's like that, and int & str are already present when type checking for Deque

Reply:
This is not necessary as `mypy` can infer the type.

Reply:
I am afraid `mypy` is specifically showing this as an error, and also that instead of Deque which is deprecated, deque can be used, which has been changed as well